storage/src/adb/immutable/mod.rs

```diff
@@ -90,18 +89,18 @@ pub struct Immutable<E: RStorage + Clock + Metrics, K: Array, V: Codec, H: CHash
     /// A log of all operations applied to the db in order of occurrence. The _location_ of an
     /// operation is its order of occurrence with respect to this log, and corresponds to its leaf
     /// number in the MMR.
-    log: VJournal<E, Variable<K, V>>,
+    log: variable::Journal<E, Variable<K, V>>,
 
     /// The number of operations that have been appended to the log (which must equal the number of
     /// leaves in the MMR).
     log_size: u64,
 
     /// The number of items to put in each section of the journal.
-    log_items_per_section: u64,
+    log_items_per_section: NonZeroU64,
```

Comment:
I prefer leaving as u64 internally to avoid having to .get() everywhere but YMMV. (Utility of the NonZero types seems mostly in configuration.)

Reply:
reverted back to u64